javascript/javascript1/week1/app.js

```diff
@@ -0,0 +1,78 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1987;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear;
+let shouldShowResultInDogYears = true;
+let resultInHumanYears;
+if (dogYear) {
+    shouldShowResultInDogYears = (dogYearFuture - dogYearOfBirth) * 7;
+    console.log(
+        "Your dog will be " + shouldShowResultInDogYears + " dog years old in 2027"
+    );
+} else {
+    resultInHumanYears = dogYearFuture - dogYearOfBirth;
+
+    console.log(
+        "Your dog will be " + resultInHumanYears + " human years old in 2027"
+    );
+}
+
+//Housey pricey (A house price estimator)
+let peterValue = [8, 10, 10, 100, "Peter"];
+
+let juliaValue = [5, 11, 8, 70, "Julia"];
+
+let housePrisePeters =
+    peterValue[0] * peterValue[1] * peterValue[2] * 2.5 * 1000 +
+    peterValue[3] * 300;
+let housePriseJulias =
+    juliaValue[0] * juliaValue[1] * juliaValue[2] * 2.5 * 1000 +
+    juliaValue[3] * 300;
+
+console.log(
+    peterValue[4] +
+    " price for house will be " +
+    housePrisePeters +
+    " and " +
+    juliaValue[4] +
+    " price for house will be " +
+    housePriseJulias
+);
```

Comment:
Great work with the Arrays. You have a super good understanding of the indexes and how to retrieve data from the arrays. 👌💫

**There are a few more steps and then you're all the way there!** 
We know that Peter's house _cost_ is 2.500.000,- and Julia's house _cost_ is 1.000.000. 
You have calculated the house _price_ for Peter to  2030000 and Julias house _price_ to 1121000.

We want the code to tell Peter and Julia whether they have payed too much for their house (if the house cost is higher than the house price) or if they got a great price on their house (if the house cost is lower than the house price). How could you add this to the code you have already written? 🤔💡